app/components/postBox/style.ts

```
@@ -84,13 +84,14 @@ export const TagAndDateBox = styled.div`
 
 export const Tag = styled.span`
   font-weight: 600;
-  font-size: ${({ theme }) => theme.fontSize.base};
+  font-size: ${({ theme }) => theme.fontSize.sm};
   color: ${({ theme }) => theme.colors.green600};
 `;
 
 export const Date = styled.span`
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)